pkg/command/utils/util.go

```diff
@@ -72,3 +72,22 @@ func GenerateJSONFile(jsonData []byte, targetJSON string) error {
 	}
 	return nil
 }
+
+func CheckOutputLocation(outputLocation string) (string, error) {
+	dirInfo, err := os.Stat(outputLocation)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return outputLocation, fmt.Errorf("output location (%s) is not existed: %s\n", outputLocation, err)
+		} else {
+			return outputLocation, fmt.Errorf("output location (%s) has error: %s\n", outputLocation, err)
```

Comment:
Which scenario will this error occur in?

Reply:
I can not reproduce this.

Reply:
If it cannot be reproduced, I think it's better to drop it to increase UT coverage. Just a tiny suggestion~
```
func CheckOutputLocation(outputLocation string) (string, error) {
	dirInfo, err := os.Stat(outputLocation)
	if err != nil {
		return outputLocation, fmt.Errorf("output location (%s) has error: %s\n", outputLocation, err)
	}
```